docs/developers/mobile.md

```diff
@@ -0,0 +1,34 @@
+# Mobile Development
+
+## Testing the client on a mobile device
+
+If you have made changes to the client that could affect the mobile experience, it is a good idea to test them on a real device. Such changes should ideally be tested with at least current versions of iOS Safari and Chrome for Android.
+
+1. Make sure your development system and mobile device are on the same local network.
+1. Get the IP address or host name of your development system (`$HOSTNAME` in the steps below). You can do this using the `hostname` terminal command on Mac/Linux.
```

Comment:
Hmm, I like your replacement _Tip_, by considering hostname as a docs placeholder you've been able to keep it much shorter. I do think it's good to tell them how to test, at this step, whether they have the hostname right (by visiting h from their phone), as things stand the hostname could be wrong but they wouldn't know it until step 6 (by which time the problem could be a number of other things as well). I'd have left in the http://robs_laptop..local:5000 example as well, examples always make instructions clearer and easier to read, even a simple instruction to append something. But +1 on the conciseness though. (And I'm happy to approve it as-is, too)